Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -48,6 +50,7 @@ struct PlaceUtilities {
                 }
                 if placeIndex != nil {
                     let place = sortedByDistance[placeIndex!]
+                    Place.travelTimesCache[place.id] = CachedTravelTime(Deferred(filledWith: DatabaseResult.succeed(value: time)), location)
```

Comment:
`flatMap` comes from a functional background so it's a little unexpected for it to have side effects but if this is what is simplest, so be it.